Gradle migration broke JPMS support

MAINFEST.MF from Maven build (`teavm-jso-apis-0.7.0-dev-1209.jar`):
```
Manifest-Version: 1.0
Created-By: Maven JAR Plugin 3.3.0
Build-Jdk-Spec: 17
Automatic-Module-Name: org.teavm.jso.apis
```
New MAINFEST.MF from Gralde build (`teavm-jso-apis-0.9.0-dev-13.jar`):
```
Manifest-Version: 1.0
```

This bug prevents usage of TeaVM with JPMS (Java-Modules):
```
t mmm-marshall-tvm-xml: Compilation failure: Compilation failure:
[ERROR] D:\projects\mmm\workspaces\main\marshall\impl\tvm-xml\src\main\java\module-info.java:[14,31] error: module not found: org.teavm.jso
[ERROR] D:\projects\mmm\workspaces\main\marshall\impl\tvm-xml\src\main\java\module-info.java:[16,35] error: module not found: org.teavm.jso.apis
```

Breaking change from commit [8014c03](https://github.com/konsoletyper/teavm/commit/8014c038c0090540bda5e9f294904f72c4a11edb)

--- a/jso/apis/build.gradle.kts
+++ b/jso/apis/build.gradle.kts
@@ -34,3 +34,9 @@ dependencies {
 teavmPublish {
     artifactId = "teavm-jso-apis"
 }
+
+tasks.withType<Jar> {
+    manifest {
+        attributes["Automatic-Module-Name"] = "org.teavm.jso.apis"
+    }
+}

